src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverter.cs

```diff
@@ -79,7 +79,5 @@ internal bool ShouldFlush(Utf8JsonWriter writer, ref WriteStack state)
         internal virtual bool ConstructorIsParameterized => false;
 
         internal ConstructorInfo ConstructorInfo { get; set; } = null!;
```

Comment:
Why is this defaulted to null? and shouln't this be `ConstructorInfo?`? 
Isn't this going to be `null` for structs that does not contain custom cunstructors anyway?